recipes/corsikaio/meta.yaml

```diff
@@ -0,0 +1,52 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "corsikaio" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0ffbcc6449ec24b340dba5f167c2aa0ce368d764ee611aa8c4c26d525a510b25
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+    - pytest-runner
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - corsikaio
+
+about:
+  home: https://github.com/fact-project/pycorsikaio
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Reader for corsika binary output files using numpy'
+
+  description: |
+    Library for reading the CORSIKA binary output files into meaningful
+    python data structures using numpy.
+  doc_url: https://github.com/fact-project/pycorsikaio
+  dev_url: https://github.com/fact-project/pycorsikaio
+
+extra:
+  recipe-maintainers:
+    - maxnoe
+    - kosack
```

Comment:
@kosack do you agree to maintain this recipe?

Reply:
yes I agree